Uplift/Views/GymDetail/Facilities/CourtCollectionViewCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  CourtCell.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 11/3/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class CourtCollectionViewCell: UICollectionViewCell {
+
+    // MARK: Private view vars
+    private let courtTitleLabel = UILabel()
+    private let courtImageView = UIImageView(image: UIImage(named: ImageNames.court))
+    private let infoLabel = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        // Views
+        infoLabel.numberOfLines = 0
+        courtTitleLabel.font = ._14MontserratSemiBold
+        addSubview(courtTitleLabel)
+        addSubview(courtImageView)
+        addSubview(infoLabel)
+
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupConstraints() {
+        let imageOffset = 9
+        let imageSize = CGSize(width: 125, height: 166)
+
+        courtTitleLabel.snp.makeConstraints { make in
+            make.top.centerX.equalToSuperview()
+        }
+
+        courtImageView.snp.makeConstraints { make in
+            make.size.equalTo(imageSize)
+            make.top.equalTo(courtTitleLabel.snp.bottom).offset(imageOffset)
+        }
+
+        infoLabel.snp.makeConstraints { make in
+            make.center.equalTo(courtImageView)
+        }
+    }
+
+    /// DailyHours should only contain hour information for the day of this cell
+    func configure(facilityHours: FacilityHoursRange, dailyGymHours: [DailyGymHours]) {
+        setCourtTitleLabel(facilityHours: facilityHours)
+        setInfoLabel(facilityHours: facilityHours, dailyHours: dailyGymHours)
+    }
+
+    // MARK: - String Response Parsing
+    private func setCourtTitleLabel(facilityHours: FacilityHoursRange) {
+        let restrictions = facilityHours.restrictions
+        if let index = restrictions.firstIndex(of: ":") {
+            courtTitleLabel.text = String(restrictions[..<index])
+        } else { // Default: Court
+            courtTitleLabel.text = "Court"
+        }
+    }
+
+    private func setInfoLabel(facilityHours: FacilityHoursRange, dailyHours: [DailyGymHours]) {
+        // Attributed Text
+        let style = NSMutableParagraphStyle()
+        style.lineSpacing = 4
+        style.alignment = .center
+        let sportAttributes: [NSAttributedString.Key: Any] = [
+            .font: UIFont._14MontserratBold as Any,
+            .paragraphStyle: style,
+            .foregroundColor: UIColor.primaryBlack
+        ]
+        let timeAttributes: [NSAttributedString.Key: Any] = [
+            .font: UIFont._12MontserratMedium as Any,
+            .paragraphStyle: style,
+            .foregroundColor: UIColor.primaryBlack
+        ]
+
+        let text = NSMutableAttributedString(string: "\(getSportName(facilityHours: facilityHours))\n", attributes: sportAttributes)
```

Comment:
put this on a line before:
`let sportName = getSportName(facilityHours: facilityHours)`